library/core/src/hint.rs

```diff
@@ -154,7 +154,7 @@ pub fn spin_loop() {
 /// [`std::convert::identity`]: crate::convert::identity
 #[cfg_attr(not(miri), inline)]
 #[cfg_attr(miri, inline(never))]
-#[unstable(feature = "test", issue = "50297")]
+#[unstable(feature = "bench_black_box", issue = "64102")]
```

Comment:
Hm, shouldn't this not use bench in the name? Is black_box taken?

Reply:
This is the name of [the RFC](https://github.com/rust-lang/rfcs/blob/master/text/2360-bench-black-box.md).